### PR TITLE
Add Replicate demo and API

### PR DESCRIPTION
**Pull request by chenxwh**
Hi @zRzRzRzRzRzRzR,

Great work on `CogVideoX`!

This pull request makes it possible to run `CogVideoX` on Replicate (https://replicate.com/chenxwh/cogvideox-t2v, https://replicate.com/chenxwh/cogvideox-i2v). Also we'd like to transfer the demo pages to `THUDM` so you can make modifications easily, and happy to help maintain/integrate the upcoming changes and improving the demo :)

### CODE REVIEW

File: README.md
Comment (zRzRzRzRzRzRzR) [inline README.md:1]: I think it should be placed in tools, not in the readme on the homepage. We can add a news item in news. Can you modify both the Chinese and Japanese versions together?

**Review by zRzRzRzRzRzRzR — COMMENTED** [review]
I think this is a very good tool, placing it at https://github.com/THUDM/CogVideo/tree/main/tools and creating a separate folder is a better choice. Would you be willing to make such a modification?

**Comment by chenxwh** [comment]
@zRzRzRzRzRzRzR thanks for the suggestion, I have made the corresponding modifications :)

File: README.md
Comment (chenxwh) [reply README.md:1]: Modified the main Readme

**Pull request merged**